test/Microsoft.ML.Predictor.Tests/TestTransposer.cs

```diff
@@ -148,19 +148,19 @@ public void TransposerTest()
             ArrayDataViewBuilder builder = new ArrayDataViewBuilder(Env);
 
             // A is to check the splitting of a sparse-ish column.
-            var dataA = GenerateHelper(rowCount, 0.1, rgen, () => (DvInt4)rgen.Next(), 50, 5, 10, 15);
-            dataA[rowCount / 2] = new VBuffer<DvInt4>(50, 0, null, null); // Coverage for the null vbuffer case.
+            var dataA = GenerateHelper(rowCount, 0.1, rgen, () => (int)rgen.Next(), 50, 5, 10, 15);
+            dataA[rowCount / 2] = new VBuffer<int>(50, 0, null, null); // Coverage for the null vbuffer case.
             builder.AddColumn("A", NumberType.I4, dataA);
             // B is to check the splitting of a dense-ish column.
             builder.AddColumn("B", NumberType.R8, GenerateHelper(rowCount, 0.8, rgen, rgen.NextDouble, 50, 0, 25, 49));
             // C is to just have some column we do nothing with.
-            builder.AddColumn("C", NumberType.I2, GenerateHelper(rowCount, 0.1, rgen, () => (DvInt2)1, 30, 3, 10, 24));
+            builder.AddColumn("C", NumberType.I2, GenerateHelper(rowCount, 0.1, rgen, () => (short)1, 30, 3, 10, 24));
             // D is to check some column we don't have to split because it's sufficiently small.
             builder.AddColumn("D", NumberType.R8, GenerateHelper(rowCount, 0.1, rgen, rgen.NextDouble, 3, 1));
             // E is to check a sparse scalar column.
             builder.AddColumn("E", NumberType.U4, GenerateHelper(rowCount, 0.1, rgen, () => (uint)rgen.Next(int.MinValue, int.MaxValue)));
             // F is to check a dense-ish scalar column.
-            builder.AddColumn("F", NumberType.I4, GenerateHelper(rowCount, 0.8, rgen, () => (DvInt4)rgen.Next()));
+            builder.AddColumn("F", NumberType.I4, GenerateHelper(rowCount, 0.8, rgen, () => (int)rgen.Next()));
```

Comment:
>(int)rgen.Next() [](start = 92, length = 16)

As far as I see, `Random.Next` already returns an integer.

Reply:
You are right.

---
In reply to: [212050999](https://github.com/dotnet/machinelearning/pull/683#discussion_r212050999) [](ancestors = 212050999)